pythonosc/osc_packet.py

```diff
@@ -68,7 +68,14 @@ def __init__(self, dgram: bytes) -> None:
                     key=lambda x: x.time,
                 )
             elif osc_message.OscMessage.dgram_is_message(dgram):
-                self._messages = [TimedMessage(now, osc_message.OscMessage(dgram))]
+                self._messages = []
+                while(True):
```

Comment:
you could also base the condition of that while loop on the presence of the msg.dgram_tail, it'd read better, something like:

```
while msg.dgram_tail:
   msg = osc_message.OscMessage(dram)
   self._messages.append(TimedMessage(now, msg)
```